src/js/fc.js

```diff
@@ -333,6 +334,10 @@ var FC = {
             gyroUse32kHz:               0,
         };
 
+        RC_SMOOTHING_CONFIG = {
+            type: 0
+        };
```

Comment:
I suppose this is left from an older version?

Reply:
This was something I added before realizing this was part of the RX_CONFIG.  I'll make sure to pull it back out, great catch!